core/cat/factory/llm.py

```diff
@@ -253,7 +286,8 @@ def get_allowed_language_models():
         LLMHuggingFaceTextGenInferenceConfig,
         LLMAzureOpenAIConfig,
         LLMAzureChatOpenAIConfig,
-        LLMOllamaConfig
+        LLMOllamaConfig,
+        LLMGeminiChatConfig
```

Comment:
just a suggestion, to better track meaningful changes, close the line with a comma, so that, next time, the change is only on one line